src/redux/raw/actions.js

```diff
@@ -0,0 +1,31 @@
+
+import createFetchAction from '../createFetchAction';
+import { shouldFetch } from '../shared/shouldFetch';
+
+
+function rawShouldFetch(state) {
+  return shouldFetch(state);
+}
+
+// ---------------------
+// Fetch Raw Tests
+// ---------------------
+const rawTestsFetch = createFetchAction({
+  typePrefix: 'raw/tests/',
+  key: 'RAW',
+  args: [],
+  shouldFetch(state) {
+    const rawTestsState = state.raw.tests;
+    return rawShouldFetch(rawTestsState);
```

Comment:
Could just call shouldFetch() directly here instead of defining rawShouldFetch = shouldFetch above?